src/deepsparse/transformers/utils/token_generator.py

```diff
@@ -77,16 +77,17 @@ def generate(self, logits: numpy.ndarray) -> numpy.ndarray:
         :param logits: the logits from the model with shape (vocab_size,)
         :return: the sampled token
         """
-        if self.top_k:
-            logits = self.apply_top_k(logits)
-        if self.top_p:
-            logits = self.apply_top_p(logits)
-
```

Comment:
Seems to be the same case for v2?

Reply:
Yes. This fix makes the e2e tests for v2 green :)

Reply:
Nice!